src/hooks/usePrivateRoomMonitor.ts

```
@@ -100,11 +101,17 @@ export default function usePrivateRoomMonitor(
         setRoller(dice);
         setRoomBackground(roomBackgroundURL || '');
 
+        // Only rebuild if the room message is recent (within last 5 minutes)
+        const messageDate = parseMessageTimestamp(roomMessage.timestamp);
+        const messageAge = messageDate ? Date.now() - messageDate.getTime() : Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)